### PR TITLE
VDS-335: Assign UserGroup to customer segment

### PR DESCRIPTION
**Pull request by ilyawzrd**
Снимок экрана 2020-12-03 в 18 15 31

### CODE REVIEW

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Core/Models/DemoCustomerSegment.cs [hunk 1]
```diff
@@ -15,6 +15,8 @@ public DemoCustomerSegment()
 
         public string Description { get; set; }
 
+        public string UserGroup { get; set; }
```
Comment (asvishnyakov) [inline src/VirtoCommerce.DemoCustomerSegmentsModule.Core/Models/DemoCustomerSegment.cs:18]: [StringLength(64)]
https://github.com/VirtoCommerce/vc-module-customer/blob/ff7dd139fcbb8a9ead90c988d141d5dadec7cd0e/src/VirtoCommerce.CustomerModule.Data/Model/MemberGroupEntity.cs#L9

**Review by asvishnyakov — CHANGES_REQUESTED** [review]
All other changes is OK

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Core/Models/DemoCustomerSegment.cs (hunk 1 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Core/Models/DemoCustomerSegment.cs:18]: Oh, sorry, it's about Data model, of course

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Data/Models/DemoCustomerSegmentEntity.cs [hunk 2]
```diff
@@ -9,16 +9,18 @@
...
         [Required]
         [StringLength(128)]
         public string Name { get; set; }
-        
+
         [Required]
         [StringLength(1024)]
         public string Description { get; set; }
 
+        public string UserGroup { get; set; }
```
Comment (asvishnyakov) [inline src/VirtoCommerce.DemoCustomerSegmentsModule.Data/Models/DemoCustomerSegmentEntity.cs:22]: [StringLength(64)]
https://github.com/VirtoCommerce/vc-module-customer/blob/ff7dd139fcbb8a9ead90c988d141d5dadec7cd0e/src/VirtoCommerce.CustomerModule.Data/Model/MemberGroupEntity.cs#L9

**Review by asvishnyakov — CHANGES_REQUESTED** [review]
All other changes is OK

File: src/VirtoCommerce.DemoCustomerSegmentsModule.Data/Models/DemoCustomerSegmentEntity.cs (hunk 2 above)
Comment (ilyawzrd) [reply src/VirtoCommerce.DemoCustomerSegmentsModule.Data/Models/DemoCustomerSegmentEntity.cs:23]: done

**Review by trueboroda — APPROVED** [review]

**Review by pushnitsa — DISMISSED** [review]

**Review by trueboroda — APPROVED** [review]

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**